piksi_tools/console/utils.py

```diff
@@ -638,4 +639,9 @@ def loop():
 resource_stream = partial(pkg_resources.resource_stream, 'piksi_tools')
 icon = ImageResource(resource_filename('console/images/icon.png'))
 
-swift_path = os.path.normpath(os.path.join(os.path.expanduser("~"), 'SwiftNav'))
+home_path = os.path.expanduser("~" + os.sep)
+loc = locale.getdefaultlocale()
```

Comment:
storing the result as the decomposed tuple might be more clear here ([docs](https://docs.python.org/2/library/locale.html#locale.getdefaultlocale)):
```suggestion
(language_code, encoding) = locale.getdefaultlocale()
```
and explicitly testing for `None` to indicate that `encoding` is not a boolean